src/scripts/itemDiscuss/dimItemDiscuss.directive.template.html

```diff
@@ -0,0 +1,82 @@
+<div id="discuss-drawer" ng-if="vm.show">
+    <div class="discuss-options">
+        <button class="dim-button" style="float: right;" ng-click="vm.cancel()" translate="Loadouts.Cancel"></button>
```

Comment:
eutral has a pending PR based on top of the UI I've put together that makes it way less awful than I did, so I'm deferring CSS/HTML fixes until I get a hold of that.